gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -311,4 +329,31 @@ class BaselineIdea extends AbstractBaselinePlugin {
             module.dependencies.addAll(projectRefs)
         }
     }
+
+    /**
+     * Configures some defaults on the save-actions plugin, but only if it hasn't been configured before.
+     */
+    private static void configureSaveActions(Node rootNode) {
+        GroovyXmlUtils.matchOrCreateChild(rootNode, 'component', [name: 'SaveActionSettings'], [:]) {
+            // Configure defaults if this plugin is configured for the first time only
```

Comment:
is this to allow users to override the configuration?

Reply:
yes, or they might have configured this plugin already for this project and then we don't want to override their configuration